R/conversion.R

```diff
@@ -70,7 +70,7 @@ cell_size <- function(col_rel_width, col_total_width) {
 #'
 #' @noRd
 convert <- function(text,
-                    load_stringi = class(try(stringi::stri_replace_all_fixed, silent = TRUE)) != "try-error") {
+                    load_stringi = requireNamespace("stringi", quietly = TRUE)) {
```

Comment:
BTW, the safer way to test if an object has a certain class is `inherits(x, className)` instead of `class(x) == className`, because `class()` could be of length > 1.